mobile/src/components/uam/groups-table/groups-table.tsx

```diff
@@ -29,14 +29,16 @@ const GroupsTable: FC = () => {
     action: <ActionCell id={group.id} onDelete={handleGroupsItemDelete} />,
   }));
 
-  useEffect(() => {
-    dispatch(
-      uamActions.getGroups({
-        page,
-        count: PaginationDefaultValue.DEFAULT_COUNT,
-      }),
-    );
-  }, [page]);
+  useFocusEffect(
+    useCallback(() => {
+      dispatch(
+        uamActions.getGroups({
+          page,
+          count: PaginationDefaultValue.DEFAULT_COUNT,
+        }),
+      );
```

Comment:
useCallback will do nothing in this case

Reply:
Hm... but how to add something to dependencies - or instead useFocusEffect  use inside UseEffect  `navigation.addListener('focus', () => { }); `

Reply:
oh, ok, I see